utils/mlir2FileCheck.py

```diff
@@ -346,6 +356,7 @@ def main(argv):
 
     # Process the input.
     curr_parallel_color = -1
+    print("\n// " + input_command)
```

Comment:
Debug?

Reply:
Good catch, I actually wanted this, because if I have to regenerate the file sometime later, I have the naming option that I used saved as a comment. If you don't want it, then one can just copy the FileCheck automatically generated string after that comment, which is placed first (so easy to skip)